mysql-test/suite/galera/r/MW-258.result

```diff
@@ -21,6 +22,8 @@ SHOW VARIABLES LIKE 'wsrep_desync';
 Variable_name	Value
 wsrep_desync	OFF
 UNLOCK TABLES;
+SET GLOBAL wsrep_provider_options = 'signal=wsrep_desync_left_local_monitor';
```

Comment:
We should use script instead.
--source include/galera_clear_sync_point.inc
--source include/galera_signal_sync_point.inc